content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/aging.rst

```diff
@@ -1,98 +1,224 @@
+  company by which this stock move took place.
+- :guilabel:`Remaining Quantity`: the number of units remaining for this valuation of the product,
+  after demand has been accounted for (even from other stock moves). This field can be especially
+  helpful for :abbr:`FIFO (First In, First Out)` and :abbr:`AVCO (Average Cost)` accounting, as it
+  conveys which units of stock came into a warehouse first and the value of said stock.
+- :guilabel:`Unit Value`: the cost of one unit of the product for the company (**not** the price to
+  consumers).
+- :guilabel:`Description`: a description of the reason for this stock valuation (typically, a stock
+  move has occurred). By default, this field is set as the concatenation of the :guilabel:
+  `Reference` and :guilabel:`Product` fields. However, the field may also display other important
+  messages for this line item, such as a note stating that the line item is an adjustment due to a
+  change in the product's inventory valuation method.
+- :guilabel:`Remaining Value`: the value of this product's current stock levels for this particular
+  stock move, after demand has been accounted for. Along with :guilabel:`Remaining Quantity`, this
+  field can be especially helpful for :abbr:`FIFO (First In, First Out)` and
+  :abbr:`AVCO (Average Cost)` accounting, as they convey which stock came into a warehouse first and
+  the value of said stock.
+
+.. tip::
+   Some of these settings may not appear unless first enabled in Odoo's **:doc:`Settings
+   <../../../general>`** application.
+
+.. image:: aging/stock-valuation-report.png
+   :alt: Stock valuation report.
+
+.. _inventory/warehouses_storage/svl:
+
+Stock valuation layers (SVLs)
+-----------------------------
+
+Each line item in the :guilabel:`Stock Valuation` report represents a record in Odoo's system known
+as a *stock valuation layer (SVL)*. :abbr:`SVLs (stock move layers)` are generated when products
+move in a way that impacts their stock valuation. Specifically, the stock moves that generate
+:abbr:`SVLs (stock move layers)` are warehouse receipts, deliveries, dropshipping orders, and
+dropshipping returns. These stock moves must first be validated (by clicking the
+:guilabel:`Validate` button) for the :abbr:`SVL (stock move layer)` to be created.
+
+If a product's inventory valuation method changes on the product form, new line items are generated
+on the :guilabel:`Stock Valuation` report to reflect the resulting :abbr:`SVLs (stock move layers)`.
+For example, if the valuation method changes from *standard price* to either *:abbr:`AVCO (Average
+Cost)`* or *:abbr:`FIFO (First In, First Out)`* accounting, *revaluation entries* will be
+automatically posted to reflect the change in pricing for goods that remain in stock. One entry will
+be negative to "remove" the old pricing, and the second entry will be positive to record the new
+pricing. These entries are connected to journal entries in Odoo's **Accounting** app.
```

Comment:
Syntax error: you can't stack formatting (italics, bold, etc) on RST roles (abbr, guilabel, menuselection, doc, etc). Please avoid this :)
```suggestion
For example, if the valuation method changes from *standard price* to either :abbr:`AVCO (Average
Cost)` or :abbr:`FIFO (First In, First Out)` accounting, *revaluation entries* will be automatically
posted to reflect the change in pricing for goods that remain in stock. One entry will be negative
to "remove" the old pricing, and the second entry will be positive to record the new pricing. These
entries are connected to journal entries in Odoo's **Accounting** app.
```